spaceship/lib/spaceship/test_flight/client.rb

```diff
@@ -164,18 +202,19 @@ def create_app_level_tester(app_id: nil, first_name: nil, last_name: nil, email:
       handle_response(response)
     end
 
-    def put_tester_to_group(app_id: nil, tester_id: nil, group_id: nil)
+    def put_tester_to_group(app_id: nil, tester: nil, group_id: nil)
```

Comment:
I think there was some convention around not passing in model objects into the client. But I am totally cool with this happening.

Reply:
Yeah, I think that makes sense.